### PR TITLE
implemented explicit fileformat option

### PR DESCRIPTION
**Pull request by leipzig**
This implements the explicit file format option for this tool (as opposed to autodetect based on suffixes) to deal with Cromwell's unfortunate behavior of renaming files identified by URLs

### Checklist
- [X] Pull request details were added to CHANGELOG.rst

### CODE REVIEW

File: src/biowdl_input_converter/__init__.py [hunk 1]
```diff
@@ -93,11 +104,7 @@ def samplesheet_to_json(samplesheet: Path,
 
 def main():
     args = argument_parser().parse_args()
-    output_json = samplesheet_to_json(
-        samplesheet=Path(args.samplesheet),
-        old_style_json=args.old_style_json,
-        file_presence_check=not args.skip_file_check,
-        file_md5_check=args.check_file_md5sums)
+    output_json = samplesheet_to_json(samplesheet=Path(args.samplesheet), fileformat=args.format, old_style_json=args.old_style_json, file_presence_check=not args.skip_file_check, file_md5_check=args.check_file_md5sums)
```
Comment (rhpvorderman) [inline src/biowdl_input_converter/__init__.py]: can you put the newlines back in? It is more readable that way.

**Review by rhpvorderman — CHANGES_REQUESTED** [review]
Looks good, thanks!

I just found a few formatting issues. Also, I updated the main branch just now to move the testing to github actions. Can you merge the dev branch back in?

File: src/biowdl_input_converter/__init__.py (hunk 1 above)
Comment (leipzig) [reply src/biowdl_input_converter/__init__.py]: ok done

**Comment by leipzig** [comment]
ok I've merged dev in and made the formatting changes

**Review by rhpvorderman — APPROVED** [review]

**Comment by rhpvorderman** [comment]
Looks good! thanks for adding a test as well.

**Comment by leipzig** [comment]
Thanks for the merge. I'm determined to get all of BioWDL loaded onto Truwl.com so I'll probably be in touch!

**Comment by rhpvorderman** [comment]
@leipzig Thanks for the update. I  created a new version which was uploaded into bioconda just now. A new image on quay.io should be available shortly.

**Pull request merged**